a8-versions.json

```diff
@@ -3,5 +3,5 @@
   "mainClass": "a8.versions.apps.Main",
   "organization": "a8",
   "artifact": "a8-versions_2.12",
-  "version": "1.0.0-20180425_1229_master"
```

Comment:
Changed the a8-versions.json to not be pegged at a version but use the latest <master> code with the help updates.